pasta_eln/database.py

```diff
@@ -542,7 +541,7 @@ def getView(self, thePath:str, startKey:Optional[str]=None, preciseKey:Optional[
       else:
         res = list(v.result)
     except Exception:
-      print('**ERROR dgv01: Database / Network problem for path |',thePath[0],thePath[1])
+      print('**ERROR dgv01: Database / Network problem for path |', thePath)
```

Comment:
I think, it would be good to log all the errors instead of doing console prints, but this could be refactored later too..